app/models/container.rb

```diff
@@ -43,4 +43,11 @@ def event_where_clause(assoc = :ems_events)
   def perf_rollup_parents(_interval_name = nil)
     # No rollups: nodes performance are collected separately
   end
+
+  def disconnect_inv
+    _log.info "Disconnecting Container [#{name}] id [#{id}] from EMS [#{ext_management_system.name}]" \
+    "id [#{ext_management_system.id}] "
+    self.deleted_on = Time.now.utc
```

Comment:
@moolitayer to be future-proof we should add `old_ems_id` here as well. Also add a `FIXME` about setting `ems_id` to nil.